diff --git a/src/utils/screengrabber.cpp b/src/utils/screengrabber.cpp
index 062c4bde..d313f940 100644
--- a/src/utils/screengrabber.cpp
+++ b/src/utils/screengrabber.cpp
@@ -141,10 +141,6 @@ QPixmap ScreenGrabber::grabEntireDesktop(bool& ok)
                   "activate the grim-based general wayland screenshot adapter");
                 freeDesktopPortal(ok, res);
 #else
-                AbstractLogger::warning()
-                  << tr("grim's screenshot component is implemented based on "
-                        "wlroots, it may not be used in GNOME or similar "
-                        "desktop environments");
                 generalGrimScreenshot(ok, res);
 #endif
                 break;
